Google/secret-manager/README.md

```diff
@@ -0,0 +1,70 @@
+# Google Secret Manage Module  
+This module can be used to manage Secret Manager Secrets and their IAM policies in a project (Note: this module does not manage the secret versions, i.e. the data).
+This is created by defining a Terraform `module` which references a `yaml` configuration file (see [configuration](#google-project-basic-configuration).
+e.g. `main.tf`:
+```terraform
+module dev_projects {
+  source      = "github.com/mesoform/Multi-Cloud-Platform-Foundations//Google/secret-manager"
+  secrets_yml = "./secrets.yaml"
+}
+```
+## Prerequisites
+### Required Services
+The following services must be enabled in the project the secrets will be deployed to:
+* `secretmanager.googleapis.com`
+### Required Permissions
+The account used for creating the secret will need to have the following roles:
+ * `roles/secretmanager.admin` for creating secrets and managing the IAM policy (note: this will also give the service 
```

Comment:
If we were to define a secretmanager.editor custom role, could we create one which can't access the secret data but can still set up the secrets?